nano/node/gap_cache.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <nano/lib/numbers.hpp>
+#include <nano/lib/utility.hpp>
+#include <nano/secure/common.hpp>
```

Comment:
Is the secure include needed in the header?

Reply:
Yep